_docs/_latest/configure/general/non-org-root.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Running Forseti from Non-Organization Root
+order: 100
+---
+
+# {{ page.title }}
+
+By default, Forseti is designed to be installed with complete
+organization access, and run with the organization as the root node in the
+resource hierarchy.
+
+But, you also have the option to run Forseti even if you only own a subset
+of resources, such as a specific folder, or projects that are directly under
+the organization. Inventory and Scanner will be supported for use on these
+subset of resources, but Explain will not be supported.
+
+## How to Install and Configure
+
+Follow the process below:
+
+   1. Run the Forseti [installer]({% link _docs/latest/setup/install.md %}).
+   The installer will try to assign org-level roles, but you can safely ignore
+   this.
+   1. If you want Forseti to run from a folder, edit `forseti_conf_server.yaml`
+   and point the `root_resource_id` to the target folder:
+   `folders/<foo_folder_id>`. Grant the Forseti server service account to have
+   the same roles on the target folder, as was [originally granted on the
+   organization]({% link _docs/latest/concepts/service-accounts.md %}).
+   1. If you want Forseti to run on projects that you own, leave the
+   `root_resource_id` pointed to the organization. Grant project
```

Comment:
If the user who ran the installation has org admin access, this will scan for all the projects?